tests/functional/navigator/test_navigator.py

```diff
@@ -131,17 +131,6 @@ def test_navigate_until_text(self):
                                                        "About",
                                                        screen_change_before_first_instruction=False)
 
-    def test_navigate_until_text_cannot_find_text(self):
```

Comment:
Why the removal ?

Reply:
I have put it in the first commit (300sec test which is also done in a 5sec format...)
```
[tests: Remove 300sec tests](https://github.com/LedgerHQ/ragger/pull/110/commits/bbb0970cb36d74ed6d67f778f8504748abc2391c)

The functionnality is already tested in following test:
test_navigate_until_text_screen_change_timeout().
```

Reply:
The timeout is configurable so we can set it to 3sec and keep the test if it's the only reason
If this test is a needless duplicate let's scratch it

Reply:
there is already a duplicate with timeout set to 5sec, so let's scratch this version.